### PR TITLE
Add particles

### PR DESCRIPTION
**Pull request by brentru**
Brings Adafruit_Sensor in-line with https://github.com/adafruit/Wippersnapper_Protobuf/blob/master/proto/wippersnapper/i2c/v1/i2c.proto by adding support for PM10, PM25, PM100. Readings in ppm.

Resolves https://github.com/adafruit/Adafruit_Sensor/issues/40

### CODE REVIEW

**Comment by brentru** [comment]
@tyeth Does this pull request address #46 in the way you also wanted?

File: Adafruit_Sensor.cpp [hunk 1]
```diff
@@ -73,9 +73,27 @@ void Adafruit_Sensor::printSensorDetails(void) {
...
     Serial.print(F("Equivalent/estimated CO2 (ppm)"));
     break;
+  case SENSOR_TYPE_PM10_STD:
+    Serial.print(F("Standard Particulate Matter 1.0 (ppm)"));
+    break;
+  case SENSOR_TYPE_PM25_STD:
+    Serial.print(F("Standard Particulate Matter 2.5 (ppm)"));
+    break;
+  case SENSOR_TYPE_PM100_STD:
+    Serial.print(F("Standard Particulate Matter 100 (ppm)"));
```
Comment (tyeth) [inline Adafruit_Sensor.cpp:86]: Should be ten point zero ( 10.0 ) not one hundred

File: Adafruit_Sensor.cpp [hunk 2]
```diff
@@ -73,9 +73,27 @@ void Adafruit_Sensor::printSensorDetails(void) {
...
+    break;
+  case SENSOR_TYPE_PM10_ENV:
+    Serial.print(F("Environmental Particulate Matter 1.0 (ppm)"));
+    break;
+  case SENSOR_TYPE_PM25_ENV:
+    Serial.print(F("Environmental Particulate Matter 2.5 (ppm)"));
+    break;
+  case SENSOR_TYPE_PM100_ENV:
+    Serial.print(F("Environmental Particulate Matter 100 (ppm)"));
```
Comment (tyeth) [inline Adafruit_Sensor.cpp:95]: Should be ten point zero ( 10.0 ) not one hundred

File: Adafruit_Sensor.h [hunk 3]
```diff
@@ -147,6 +153,18 @@ typedef struct {
...
+                        (ppm) */
+    float pm25_std;  /**< Standard Particulate Matter 2.5 in parts per million
+                        (ppm) */
+    float pm100_std; /**< Standard Particulate Matter 100 in parts per million
+                        (ppm) */
+    float pm10_env;  /**< Environmental Particulate Matter 1.0 in parts per
+                        million (ppm) */
+    float pm25_env;  /**< Environmental Particulate Matter 2.5 in parts per
+                        million (ppm) */
+    float pm100_env; /**< EnvironmentalParticulate Matter 100 in parts per
```
Comment (tyeth) [inline Adafruit_Sensor.h:166]: Should be ten point zero ( 10.0 ) not one hundred

**Comment by tyeth** [comment]
@brentru In the description the PM100 value is <10.0, not 100, like the PM10 is <1.0 micron and PM25 is <2.5µm
I don't know if it's worth mentioning that these values are for upto, or less than, the value. This confuses matters because I don't know if the 4 micron value is included in the <10.0 value, which theoretically it should be for comparability with other devices that don't show <4.0 micron separately and only have a 10 micron value. pfff, don't worry for now apart from the first issue, and maybe adding less than symbol and size unit could be good, i.e. <2.5 micron or µm

**Comment by brentru** [comment]
@tyeth  Thanks for the review! I've updated `master` to reflect your feedback.

**Pull request merged**